codemods/ts-morph/next/13/replace-next-router/config.json

```diff
@@ -1,13 +1,8 @@
 {
-    "schemaVersion": "1.0.0",
-    "name": "next/13/replace-next-router",
-    "engine": "ts-morph",
-    "extensions": [
-        "js*",
-        "ts*"
-    ],
-    "dependencyVersionLowerThan": [
-        "next",
-        "13.0.0"
-    ]
+	"schemaVersion": "1.0.0",
+	"name": "next/13/replace-next-router",
+	"description": "",
```

Comment:
`Replaces all usages of the useRouter() calls with useSearchParams and usePathname"